src/h264/h264.h

```diff
@@ -180,8 +226,328 @@ class H264SeqParameterSet
 	DWORD			frame_crop_right_offset = 0;
 	DWORD			frame_crop_top_offset = 0;
 	DWORD			frame_crop_bottom_offset = 0;
-	//bool			vui_parameters_present_flag = false;
+	bool			vui_parameters_present_flag = false;
 	bool			separate_colour_plane_flag = 0;
+
+	struct HrdParameters
+	{
+		uint32_t              cpb_cnt_minus1 = 0;
+		uint32_t              bit_rate_scale = 0;
+		uint32_t              cpb_size_scale = 0;
+		std::vector<uint32_t> bit_rate_value_minus1;
+		std::vector<uint32_t> cpb_size_value_minus1;
+		std::vector<uint32_t> cbr_flag;
+		uint32_t              initial_cpb_removal_delay_length_minus1 = 0;
+		uint32_t              cpb_removal_delay_length_minus1 = 0;
+		uint32_t              dpb_output_delay_length_minus1 = 0;
+		uint32_t              time_offset_length = 0;
+		bool ParseHRDParams(RbspBitReader &r)
+		{
+			cpb_cnt_minus1 = r.GetExpGolomb();
+			if (cpb_cnt_minus1 < CpbCntMinusOneMin ||
+				cpb_cnt_minus1 > CpbCntMinusOneMax)
+			{
+				Warning("-H264SeqParameterSet::ParseHRDParameters() Invalid cpb_cnt_minus1 %u. Not in Range [%u, %u]\n", 
+							cpb_cnt_minus1, CpbCntMinusOneMin, CpbCntMinusOneMax);
+				return false;
+			}
+			bit_rate_scale = r.Get(4);
+			cpb_size_scale = r.Get(4);
+			for (uint32_t SchedSelIdx = 0; SchedSelIdx <= cpb_cnt_minus1; SchedSelIdx++)
+			{
+				bit_rate_value_minus1.push_back(r.GetExpGolomb());
+				cpb_size_value_minus1.push_back(r.GetExpGolomb());
+				cbr_flag.push_back(r.Get(1));
+			}
+			initial_cpb_removal_delay_length_minus1 = r.Get(5);
+			cpb_removal_delay_length_minus1 = r.Get(5);
+			dpb_output_delay_length_minus1 = r.Get(5);
+			time_offset_length = r.Get(5);
+			return true;
+		}
+		void DumpHrdParams() const 
```

Comment:
keep naming consistency
```suggestion
		void Dump() const 
```